test/launcher.spec.ts

```diff
@@ -268,7 +268,7 @@ describe('Launcher specs', function () {
         expect(await page2.evaluate(() => localStorage.hey)).toBe('hello');
         await browser2.close();
         // This might throw. See https://github.com/puppeteer/puppeteer/issues/2778
-        await rmAsync(userDataDir).catch(() => {});
+        await rmAsync(userDataDir).catch(() => { });
```

Comment:
It looks like there are a bunch of drive by changes, can you run `npm run eslint-fix`?

Reply:
Oh, my mistake, I will fix it. Thank you.